uber/config.py

```diff
@@ -315,6 +315,12 @@ def _is_intstr(s):
         return s[1:].isdigit()
     return s.isdigit()
 
+if not c.KIOSK_CC_ENABLED:
+    del _config['enums']['door_payment_method']['stripe']
```

Comment:
@EliAndrewC: I looked up ConfigObj and found that you can remove options from it directly, so that's what I'm doing here instead. Look good?